src/Sylius/Bundle/ApiBundle/spec/CommandHandler/Admin/RequestResetPasswordEmailHandlerSpec.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ *  This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace spec\Sylius\Bundle\ApiBundle\CommandHandler\Admin;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Sylius\Bundle\ApiBundle\Command\Admin\RequestResetPasswordEmail;
+use Sylius\Bundle\ApiBundle\Command\Admin\SendResetPasswordEmail;
+use Sylius\Calendar\Provider\DateTimeProviderInterface;
+use Sylius\Component\Core\Model\AdminUserInterface;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+use Sylius\Component\User\Security\Generator\GeneratorInterface;
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Symfony\Component\Messenger\Stamp\DispatchAfterCurrentBusStamp;
+
+final class RequestResetPasswordEmailHandlerSpec extends ObjectBehavior
+{
+    private const SAMPLE_EMAIL = 'admin@example.com';
+
+    private const SAMPLE_LOCALE_CODE = 'en_US';
```

Comment:
I understand the reason 👌 but I would go with hardcoding these values in specs body, the general rule of thumb is to make spec function readable from top to bottom, without being forced to jump to any variable/external function 🖖